src/applications/personalization/profile-2/components/ProfileInfoTable.jsx

```diff
@@ -76,7 +76,7 @@ const ProfileInfoTable = ({
             >
               {row.title}
             </h4>
-            <p className={tableRowDataClasses.join(' ')}>{row.value}</p>
+            <div className={tableRowDataClasses.join(' ')}>{row.value}</div>
```

Comment:
had to switch this to a `div` as the content passed to it contained `p`s at the root level and nesting a `p` directly in a `p` throws console errors.